services/ui-src/src/components/reports/ReportPageFooter.tsx

```diff
@@ -2,17 +2,17 @@ import { useNavigate } from "react-router-dom";
 // components
 import { Box, Button, Flex, Spinner } from "@chakra-ui/react";
 import { Icon } from "components";
-import { useUser } from "utils";
+// utils
+import { useFindRoute, useUser } from "utils";
 import { FormJson } from "types";
+import { mcparReportRoutesFlat } from "forms/mcpar";
 
-export const ReportPageFooter = ({
-  loading,
-  form,
-  previousRoute,
-  nextRoute,
-  ...props
-}: Props) => {
+export const ReportPageFooter = ({ loading, form, ...props }: Props) => {
   const navigate = useNavigate();
+  const { previousRoute, nextRoute } = useFindRoute(
+    mcparReportRoutesFlat,
+    "/mcpar"
```

Comment:
For all these "/mcpar" base path things you can get the basepath from ReportProvider's `report.basePath` i think.

Reply:
report won't have basepath on it. It'd come from an imported mcparjson object. I'm going to leave it for now